### PR TITLE
Add basic conduit guide

### LINKED ISSUE #1064: [Matrix Conduit] Add guide
[Conduit](https://conduit.rs) is a Homeserver for the Matrix messaging protocol written in Rust.

To test stuff out, I tried installing it on uberspace and thought, I would share the workflow here.

### PR DESCRIPTION
**Pull request by jfowl**
Closes #1064

### CODE REVIEW

File: source/guide_conduit.rst [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+
+Installation
+============
+
+Download the latest development build and mark it as executable:
```
Comment (nichtmax) [inline source/guide_conduit.rst:33]: You download the binary directly into the home folder, how about `~/bin`?

File: source/guide_conduit.rst [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  which contains information about where to find the actual Matrix server for that domain (``uberspace.de``).
+  
+  This is mostly practical for bigger Matrix servers with many users, where you don't want to run the website (uberspace.de) on the same hardware as the matrix server.
+  In this guide however, we will just host the Matrix server and your potential website side by side on the same uberspace account.
+
+Add .well-known files to point to the same domain:
+
+::
+
+  [isabell@stardust ~]$ mkdir -p ~/html/.well-known/matrix/
```
Comment (nichtmax) [inline source/guide_conduit.rst:161]: Please don't echo stuff into files, just tell users to edit the file like you did with the other ones above as stated in our [style guide](https://github.com/Uberspace/lab/blob/main/STYLE.md).

**Review by nichtmax — CHANGES_REQUESTED** [review]
Thanks for your guide! Looks good, just a few things to change ☺️

**Comment by jfowl** [comment]
@nichtmax thanks for the review. I plan to address your comments and polish some things up in ~ 3 weeks.

File: source/guide_conduit.rst (hunk 1 above)
Comment (jfowl) [reply source/guide_conduit.rst:33]: Done.

File: source/guide_conduit.rst (hunk 2 above)
Comment (jfowl) [reply source/guide_conduit.rst]: Done.

**Comment by nichtmax** [comment]
I fixed a few small things, thanks for the guide @jfowl! 🎉 

Get in touch via hallo@uberspace.de for your cup ☺️

**Comment by 23n** [comment]
Hi, the database path should be:
database_path = "/home/<username>/conduit_data"
not 
database_path = "/home/<username>/conduit_data/conduit.db"
otherwise the db will not be created, or am I wrong?

**Comment by jfowl** [comment]
I added a to do for me for the first week of January to fix this.

**Comment by jfowl** [comment]
@23n this was fixed by https://github.com/Uberspace/lab/pull/1195

**Pull request merged**